### PR TITLE
[Term Entry] Adding SQL operator greater than

### PR DESCRIPTION
**Pull request by selenahuang017**
Description
Adding SQL greater than operator to SQL concepts under docs/content/sql/concepts/operators/terms/greater-than/greater-than.md.

Issue Solved
Addresses issue: https://github.com/Codecademy/docs/issues/4884 

Type of Change
Adding a new entry
Checklist
[x ] All writings are my own.
[x ] My entry follows the Codecademy Docs style guide.
[x ] My changes generate no new warnings.
[x ] I have performed a self-review of my own writing and code.
[x ] I have checked my entry and corrected any misspellings.
[x ] I have made corresponding changes to the documentation if needed.
[x ] I have confirmed my changes are not being pushed from my forked main branch.
[x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
[x ] I have linked any issues that are relevant to this PR in the Issues Solved section.

### CODE REVIEW

File: content/sql/concepts/operators/terms/greater-than/greater-than.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+## Example 2
+
+Return rows where `order_date` is greater than or equal to (after) `2020-01-01`.
+
+```sql
+SELECT *
+FROM coffee
+WHERE order_date >= "2020-01-01";
```
Comment (SaviDahegaonkar) [inline content/sql/concepts/operators/terms/greater-than/greater-than.md:50]:
``` suggestion
SELECT *
FROM coffee
WHERE order_date >= '2020-01-01';
```
In SQL single quotes are very widely accepted and recommended for proper understanding and to ensure compatibility across different database systems.

**Review by SaviDahegaonkar — APPROVED** [review]
Hello @selenahuang017,
Your file is completely correct and follows all the style-guide guidelines, hence it is approved.

Thanks.

**Comment by selenahuang017** [comment]
@SaviDahegaonkar Hi! I have updated the branch with the edits and linted the file, so it is ready to merge! Thank you.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Excellent entry! Hey @selenahuang017, I have suggested some changes, please make them. Thanks!!

**Comment by selenahuang017** [comment]
Hi! I'm not sure why the format would fail when the link passed, and the run doesn't give any information on what is wrong. Can someone please check on this and let me know?

**Comment by avdhoottt** [comment]
> Hi! I'm not sure why the format would fail when the link passed, and the run doesn't give any information on what is wrong. Can someone please check on this and let me know?

I'll fix the format test. Thank you for contributing!!

**Review by avdhoottt — APPROVED** [review]
LGTM!

**Pull request merged**